### PR TITLE
docs(puppeteer-chromium): docker alpine switch to Alpine 3.10

### PR DESCRIPTION
**Pull request by vitaliytv**
without EDGE

### CODE REVIEW

File: docs/troubleshooting.md [hunk 1]
```diff
@@ -284,23 +284,22 @@ how to run this Dockerfile from a webserver running on App Engine Flex (Node).
...
-      freetype@edge \
-      harfbuzz@edge \
-      ttf-freefont@edge
+RUN apk add --no-cache \
+      chromium \
+      nss \
+      freetype \
+      harfbuzz \
+      ttf-freefont \
+      nodejs \
```
Comment (aslushnikov) [inline docs/troubleshooting.md:335]: which version node.js does it install?

File: docs/troubleshooting.md [hunk 2]
```diff
@@ -284,23 +284,22 @@ how to run this Dockerfile from a webserver running on App Engine Flex (Node).
...
-    apk add --no-cache \
-      chromium@edge=~73.0.3683.103 \
-      nss@edge \
-      freetype@edge \
-      harfbuzz@edge \
-      ttf-freefont@edge
+RUN apk add --no-cache \
+      chromium \
+      nss \
+      freetype \
```
Comment (aslushnikov) [inline docs/troubleshooting.md:330]: I think there's also a freetype-dev now as a dependency

File: docs/troubleshooting.md (hunk 1 above)
Comment (vitaliytv) [reply docs/troubleshooting.md:335]: 10.16

File: docs/troubleshooting.md (hunk 2 above)
Comment (vitaliytv) [reply docs/troubleshooting.md:330]: added, but i don't have case for check this https://github.com/jeremykerr-sp/puppeteer/commit/5582fa29df99494e0211cad448a56df9f0200ce5

**Comment by nuragic** [comment]
Yes please! Right now the chromium version on edge changed, what specified in the docs doesn't work anymore 😟

**Review by aslushnikov — APPROVED** [review]

**Comment by aslushnikov** [comment]
@vitaliytv sorry for this taking so long. May I ask you to re-baseline this atop of master and force-push it to the PR?

**Comment by vitaliytv** [comment]
@aslushnikov maybe with chromium 76?

**Review by vitaliytv — COMMENTED** [review]
> @vitaliytv sorry for this taking so long. May I ask you to re-baseline this atop of master and force-push it to the PR?

@aslushnikov done, with chromium 76

**Pull request merged**